libretro-common/media/media_detect_cd.c

```diff
@@ -65,203 +63,6 @@ static bool media_skip_spaces(const char **buf, size_t len)
    return false;
 }
 
-/* Fill in "info" with detected CD info. Use this when you have a cue file and want it parsed to find the first data track and any pregap info. */
-bool media_detect_cd_info_cue(const char *path, media_detect_cd_info_t *info)
```

Comment:
this function was unused, but it's been moved into `vfs_implementation_cdrom.c`, where it is now being used.